subsys/nrf_security/src/core/Kconfig

```diff
@@ -10,5 +10,10 @@ choice PSA_CORE
 
 config PSA_CORE_OBERON
 	bool "PSA Core implementation - Oberon"
+	select PSA_WANT_AES_KEY_SIZE_128
+	select PSA_WANT_AES_KEY_SIZE_192
+	select PSA_WANT_AES_KEY_SIZE_256
+	select PSA_WANT_RSA_KEY_SIZE_2048
+	select PSA_WANT_RSA_KEY_SIZE_3072
```

Comment:
Surely this is not correct?

Presumably this would pull in features that the user would then need to disable.

Reply:
Yes. This looks like it should be user-configurable options, not auto-set internals

PSA_WANT_XXXX is for application/protocol/module scope